jobrunner/executors/local.py

```diff
@@ -622,6 +622,17 @@ def get_output_metadata(
 
 """
 
+MAX_CSV_ROWS_MSG = """
+The file:
+
+{filename}
+
+contained {row_count} rows, which is above the limit for moderately_sensitive files of
+{limit} rows.
+
+As such, it has *not* been copied to Level 4 storage.
```

Comment:
Please could you add a message about contacting tech-support?  (See [Slack](https://bennettoxford.slack.com/archives/C069YDR4NCA/p1714560739506289?thread_ts=1714552008.581379&cid=C069YDR4NCA).)

Reply:
I had a message about contacting tech-support, and I took it out because of the conversation on the issue (about using the same process as files rejected for other reasons).